test/integration/targets/cloudscale_server/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
-cloudscale_test_flavor: flex-2
-cloudscale_test_image: debian-9
-cloudscale_test_ssh_key: |
+cloudscale_server_flavor: flex-2
+cloudscale_server_flavor_2: flex-4
+cloudscale_server_image: debian-9
+cloudscale_server_ssh_key: |
   ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDSPmiqkvDH1/+MDAVDZT8381aYqp73Odz8cnD5hegNhqtXajqtiH0umVg7HybX3wt1HjcrwKJovZURcIbbcDvzdH2bnYbF93T4OLXA0bIfuIp6M86x1iutFtXdpN3TTicINrmSXEE2Ydm51iMu77B08ZERjVaToya2F7vC+egfoPvibf7OLxE336a5tPCywavvNihQjL8sjgpDT5AAScjb3YqK/6VLeQ18Ggt8/ufINsYkb+9/Ji/3OcGFeflnDXq80vPUyF3u4iIylob6RSZenC38cXmQB05tRNxS1B6BXCjMRdy0v4pa7oKM2GA4ADKpNrr0RI9ed+peRFwmsclH test@ansible
```

Comment:
These variables should be removed. The variables from cloudscale_common should be used instead. This makes it possible to change the test flavor or image for all cloudscale tests at one place.
The `cloudscale_server_flavor_2` variable is new and not used anywhere. It can be completely removed. If you want to add a test that uses it, this should be done in the commit adding that test.
Why did you rename all the variables? This makes the diff of the tests hard to review as all lines are marked as changed while nothing really changed. I'd prefer to stay with the original variable names unless there is a compelling reason to change the names.

Reply:
addressed.